containers/ddev-webserver/ddev-webserver-base-scripts/start.sh

```diff
@@ -93,8 +93,16 @@ ls /var/www/html >/dev/null || (echo "/var/www/html does not seem to be healthy/
 # Make sure the TERMINUS_CACHE_DIR (/mnt/ddev-global-cache/terminus/cache) exists
 sudo mkdir -p ${TERMINUS_CACHE_DIR}
 
-sudo mkdir -p /mnt/ddev-global-cache/{bashhistory/${HOSTNAME},mysqlhistory/${HOSTNAME},nvm_dir/${HOSTNAME},npm,yarn/classic,yarn/berry,corepack}
+sudo mkdir -p /mnt/ddev-global-cache/{bashhistory/${HOSTNAME},mysqlhistory/${HOSTNAME},n_prefix/${HOSTNAME},nvm_dir/${HOSTNAME},npm,yarn/classic,yarn/berry,corepack}
 sudo chown -R "$(id -u):$(id -g)" /mnt/ddev-global-cache/ /var/lib/php
+
+if command -v n >/dev/null 2>&1 && [ "${N_PREFIX:-}" != "" ] && [ "${N_INSTALL_VERSION:-}" != "" ]; then
+  ln -sf /mnt/ddev-global-cache/n_prefix/${HOSTNAME} "${N_PREFIX}"
+  # try a normal install that also uses cache and try again offline if it fails
+  n install "${N_INSTALL_VERSION}" || n install "${N_INSTALL_VERSION}" --offline
```

Comment:
```suggestion
  n install "${N_INSTALL_VERSION}" || n install "${N_INSTALL_VERSION}" --offline || true
```

`n install --offline` fails if it doesn't find the version in the cache; I think we'd be better continuing.

Reply:
I thought about this, but didn't implement it.

I think it's for good, I wish we could somehow notify the user of the error here.

Maybe I could check for `node --version` somewhere later in the `ddev start`, compare it with config and show a Go warning...